packages/core-flows/src/tax/workflows/set-tax-rate-rules.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  CreateTaxRateRuleDTO,
+  ITaxModuleService,
+  TaxRateRuleDTO,
+} from "@medusajs/types"
+import {
+  StepResponse,
+  WorkflowData,
+  createStep,
+  createWorkflow,
+  transform,
+} from "@medusajs/workflows-sdk"
+import { createTaxRateRulesStep, deleteTaxRateRulesStep } from "../steps"
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+
+type WorkflowInput = {
+  taxRateId: string
+  rules: Omit<CreateTaxRateRuleDTO, "tax_rate_id">[]
+}
+
+const listRulesStep = createStep(
```

Comment:
suggestion: we usually split the workflow and the steps in different files as well as export the step in order to be reusable for a user

Reply:
should we name it `listRuleIdsStep`?